src/test/java/com/google/api/codegen/testdata/discogapic/java/java_fakebilling.v1.json.baseline

```diff
@@ -337,47 +337,179 @@ public final class BillingAccount implements Serializable {
+import com.google.common.collect.ImmutableListMultimap;
+import java.util.LinkedList;
 import java.util.List;
 import javax.annotation.Generated;
 
 @Generated("by GAPIC")
 @BetaApi
-public final class Data implements Serializable {
-  private final String key;
-  private final String value;
+public final class BillingAccountsGetHttpRequest {
+  private static final BillingAccountsGetHttpRequest DEFAULT_INSTANCE = new BillingAccountsGetHttpRequest();
+
+
+  private final ImmutableListMultimap<String, String> queryParams;
+
+  // Query parameters.
+  private String access_token;
+  private String callback;
+  private String fields;
+  private String key;
+  private String prettyPrint;
+  private String quotaUser;
+  private String userIp;
+
+  private final ImmutableListMultimap<String, String> pathParams;
+
+  // Path parameters.
+  private String name;
+
+  private BillingAccountsGetHttpRequest(
+      String name,
+      String access_token,
+      String callback,
+      String fields,
+      String key,
+      String prettyPrint,
+      String quotaUser,
+      String userIp
+      ) {
+    this.name = name;
+    this.access_token = access_token;
+    this.callback = callback;
+    this.fields = fields;
+    this.key = key;
+    this.prettyPrint = prettyPrint;
+    this.quotaUser = quotaUser;
+    this.userIp = userIp;
 
-  private Data(
+    ImmutableListMultimap.Builder pathParams = ImmutableListMultimap.builder();
+    ImmutableListMultimap.Builder queryParams = ImmutableListMultimap.builder();
 
-    String key,
-    String value
-    ) {
-    this.key = key;
-    this.value = value;
+    if (name != null) {
+      pathParams.put("name", name.toString());
+    }
+    if (access_token != null) {
+      queryParams.put("access_token", access_token.toString());
+    }
+    if (callback != null) {
+      queryParams.put("callback", callback.toString());
+    }
+    if (fields != null) {
+      queryParams.put("fields", fields.toString());
+    }
+    if (key != null) {
+      queryParams.put("key", key.toString());
+    }
+    if (prettyPrint != null) {
+      queryParams.put("prettyPrint", prettyPrint.toString());
+    }
+    if (quotaUser != null) {
+      queryParams.put("quotaUser", quotaUser.toString());
+    }
+    if (userIp != null) {
+      queryParams.put("userIp", userIp.toString());
+    }
+
+    this.pathParams = pathParams.build();
+    this.queryParams = queryParams.build();
   }
 
-  public String getKey() {
-    return key;
+  private BillingAccountsGetHttpRequest() {
+    this.queryParams = (new ImmutableListMultimap.Builder()).build();
+    this.pathParams = (new ImmutableListMultimap.Builder()).build();
   }
 
-  public String getValue() {
-    return value;
+  // A list of key-value pairs, where each key is a query parameter and the value is the value
+  // of the parameter. A key may appear multiple times, in the case of repeatable parameters.
+  public ImmutableListMultimap<String, String> getQueryParams() {
```

Comment:
We don't want to expose Guava types on the surface anywhere (either generated types or types from GAX)